pom.xml

```diff
@@ -25,9 +25,12 @@
   <properties>
     <revision>1.321</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.361.4</jenkins.version>
+    <jenkins.version>2.472</jenkins.version>
```

Comment:
Since this changes the Jenkins minimum version, it also needs to update the version of the plugin bill of materials that is included later in the file.

The dependencyManagement section that looks like this:

```
  <dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>io.jenkins.tools.bom</groupId>
            <artifactId>bom-2.361.x</artifactId>
            <version>2102.v854b_fec19c92</version>
            <scope>import</scope>
            <type>pom</type>
        </dependency>
    </dependencies>
  </dependencyManagement>
```

needs to be updated to look like this:

```
  <dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>io.jenkins.tools.bom</groupId>
            <artifactId>bom-2.462.x</artifactId>
            <version>3258.vcdcf15936a_fd</version>
            <scope>import</scope>
            <type>pom</type>
        </dependency>
    </dependencies>
  </dependencyManagement>
```

There is now a technique in the Jenkins plugin archetype that defines the property `jenkins.baseline` and then uses that property to define both `jenkins.version` and the BOM artifactId.  That would avoid these types of mistakes in the future, but will add some more changes to this pull request.

Refer to the following pull request for an example:

* https://github.com/jenkinsci/gitlab-plugin/pull/1678